services/user.service.js

```diff
@@ -186,6 +186,39 @@ class UserService extends BaseService {
 
     return response
   }
+  async fetchUserEventsAttendance(userId, eventsService) {
+    let response
+
+    try {
+      const eventsData = await eventsService.doList({})
+      if (!eventsData) {
+        return this.getSuccessResponse({},'No existing events to check')
+      }
+      const responseData = await this.addRegisteredField(userId, eventsData.data)
+
+      return this.getSuccessResponse(responseData, 'Events retrieved successfully.')
+    } catch (err) {
+      return (response = this.getErrorResponse('Error getting user information'))
```

Comment:
response is not needed here, please remove in line 190
```suggestion
      return this.getErrorResponse('Error getting user information')
```